lib/scrabble.rb

```diff
@@ -1,18 +1,48 @@
 class Scrabble
 
-  def score(word)
-    1
+  def score(guess_word)
+    guess_word_split = take_word(guess_word)
+    total_score = assign_point_values(guess_word_split)
+    p total_score
   end
 
-  def point_values
-    {
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
+
+  def take_word(word)
+    if word == " "
+      p 0
+    elsif word == '0'
+      p 0
+    elsif
+    string = word.to_s
+    array = string.chars
+    end
   end
+
+    def assign_point_values(split_word)
```

Comment:
This would only work with a five letter word.

Reply:
Yes, I was noticing that.  I ran into the same issue on the enigma project, iterating over an array so each object in the array is switched for it's has value.  I'm trying to figure it out with a .map but having problems iterating for each index position.  for example, swap letter 1 for cypher a letter 2 for cypher b letter 3 for cypher b letter 4 for cypher a.